.gitignore

```diff
@@ -26,3 +26,6 @@
 /src/**/yarn-error.log
 /tmp
 /w
+
+# ignore emacs generated files
```

Comment:
Please remove that, this should be done in a separate PR

Reply:
Or in a [local system .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)